homeassistant/components/tado/const.py

```diff
@@ -157,6 +159,15 @@
 TADO_SWING_OFF = "OFF"
 TADO_SWING_ON = "ON"
 
+HA_TO_TADO_SWING_MODE_MAP = {
```

Comment:
I think these constants belong in the climate platform as they are climate details.